docs/Learning-Environment-Design-Learning-Brains.md

```diff
@@ -0,0 +1,75 @@
+# Learning Brains
+
+The **Learning Brain** work in different phases of
+training. When training your Agents, drag the **Learning Brain** to the
+Academy's `Training Hub` and check the checkbox `Train`. When using a pretrained 
+model, just drag the Model file into the `Model` property of the **Learning Brain**.
+
+## Training Mode
```

Comment:
I know we are calling this training, but maybe we should use somewhat more agnostic language here, in case people think it is only for training?

Reply:
Since technically it is for external control.